src/plugins/shell/include/plugins/shell/shell.h

```diff
@@ -54,6 +54,8 @@ class Shell : public PluginBase {
         NO_SYSTEM, /**< @brief No system connected. */
         CONNECTION_ERROR, /**< @brief %Connection error. */
         NO_RESPONSE, /**< @brief Response does not received */
+        NOT_COMPLETED, /**< @brief Response transfer potentially not completed
```

Comment:
Isn't it equivalent to `BUSY`, like we had in other plugins? E.g. if you try to `upload_mission()` while it's already uploading/downloading, it will say `BUSY`. It may be more consistent.

Do you guys have an opinion on that?

Reply:
from [this](https://github.com/mavlink/MAVSDK-Proto/pull/112#discussion_r359965790):

Yes, this really can confuse.

BUSY - shell is busy now
NOT_COMPLETED - in response of current request was not received ESC, so we suppose that transfer was interrupted or last slices of response lost.
May be need to use more clear name (or may be refuse this enum value)?